src/article.ts

```diff
@@ -53,7 +53,16 @@ export class Article {
     return this.updateLocalImageLinks(article);
   }
 
-  public publishArticle(token: string): Promise<ArticlePublishedStatus> {
+  private fetchArticleBodyMarkdown(articleId: number): Promise<string | null> {
+    return got(`https://dev.to/api/articles/${articleId}`, {
+      json: true,
+      method: 'GET',
+    })
+      .then((x: got.Response<ArticleApi>) => x.body.body_markdown)
+      .catch(() => null);
```

Comment:
Should at least console.error here, but why not throw?

Reply:
Agree. Fixed